src/test/java/com/epam/tc/hw5/pages/DifferentElementsPageMainContainer.java

```diff
@@ -0,0 +1,57 @@
+package com.epam.tc.hw5.pages;
+
+import java.util.List;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.Select;
+
+public class DifferentElementsPageMainContainer {
+
+    private String checkboxWater = "Water";
```

Comment:
It could be moved to ENUM.

Reply:
added enum, refactored DifferentElementsPageMainContainer file using enum